HERMES.md

```diff
@@ -178,9 +192,11 @@ hermes keys add --key-name elys-relayer --chain elystestnet-1 --mnemonic-file HO
 hermes keys add --key-name juno-relayer --chain uni-6 --mnemonic-file $HOME/.hermes/juno-relayer.txt
 ```
 
+**Important**: make sure that the imported wallet address is correct. In some cases, you will need to specify the _derivation path_ by adding a flag to the command, e.g. for Injective:`--hd-path "m/44'/60'/0'/0/0"`. Find the information about the chain's derivation path at https://cosmos.directory.
```

Comment:
I was thinking on adding ethermint address type but also it requires some changes in config.toml. So maybe it's better to move this stuff to separate section? what do you think?

Reply:
Yes perhaps. I have not used ethermint addresses so far actually, unsure whether it is a frequent thing or not really. 
But feel free to reorganize as you see fit!